kassy

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+# kassy - A bash CLI tool for running kassy in docker
+# Written By: Matthew Hartstonge <matt@mykro.co.nz>
+
+# Soak in user params
+ARGS="$@"
+
+function usage() {
+  echo "usage: kassy [all][facebook][skype][slack][telegram]                   "
+  echo "             [--debug][--log][--test][--timestamp]                     "
+  echo "                                                                       "
+  echo "  Commands:                                                            "
+  echo "    cmd            Starts a bash session inside the container.         "
+  echo "                                                                       "
+  echo "  Integrations:                                                        "
+  echo "    all            Runs all integrations, not including test.          "
+  echo "    facebook       Enables integration with Facebook.                  "
+  echo "    skype          Enables integration with Skype.                     "
+  echo "    slack          Enables integration with Slack.                     "
+  echo "    telegram       Enables integration with Telegram Integration.      "
+  echo "                                                                       "
+  echo "  Debugging and Testing:                                               "
+  echo "    --debug        Turns on verbose logging.                           "
+  echo "    --log          Writes to kassy.log in /kassy folder.               "
+  echo "    --test         Runs Kassy enabled for testing.                     "
+  echo "    --timestamp    Enables timestamps in the console and log files.    "
+  echo "                                                                       "
+}
+
+# Runs kassy given a string of whitespace separated integrations
+function runKassy() {
+  cd /kassy
+  echo "Starting Kassy"'!'
+  node main.js $1
+  echo
+  echo "                                     ... You killed me ... You Monster."
+}
+
+
+# Given params will build up an array of integrations that Kassy will start on
+function main() {
+  if [ -z "${ARGS}" ]; then
+    usage
+    exit 1
+  fi
+
+  opts=()
+  # process args
+  for arg in ${ARGS}; do
+    case ${arg} in
+      # Provide a way to escape to bash *nicely*
+      'cmd')
+        bash
+        exit 0 ;;
+
+      # Integrations
+      'all')
+        opts+=('facebook skype slack telegram')
```

Comment:
I'm not sure what you mean, but I would like all command arguments to be passed straight through to Kassy.